src/uct/api/uct.h

```diff
@@ -1318,6 +1313,12 @@ ucs_status_t uct_ep_connect_to_ep(uct_ep_h ep, const uct_device_addr_t *dev_addr
  *                               remote peer.
  * @param [in]  length           Length of the private data.
  * @param [out] ep_p             Handle to the created endpoint.
+ *
+ * @return UCS_INPROGRESS        The connection to the remote peer was initiated.
```

Comment:
pls document what would UCS_OK mean (at least in theory)

Reply:
You mean for transports that can finish the connection establishment on the client side 'in place'? For which the callback won't be invoked?

Reply:
yes